cimpl/ecqm_dataelement.txt

```diff
@@ -5225,7 +5225,7 @@ ResultValue from https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1046.6/
 
 EntryElement: LiveBirthNewbornBornInHospitalDiagnoses
 Based on: Diagnoses
-Description: "(Clinical Focus: This set of values contains diagnoses that are commonly used in the inpatient setting for newborn live born, and born in hospital. This value set is a grouping value set composed of  ICD-10-CM and SNOMED CT value sets.),(Data Element Scope: The intent of this data element is to identify patients who have active diagnosis, mapped to QDM 'Diagnosis' or to the 'Principal Diagnosis' or 'Diagnosis' attributes of the QDM 'Encounter' datatype.),(Inclusion Criteria: This value set includes ICD-10-CM  and SNOMED CT value sets.),(Exclusion Criteria: None.)"
+Description: "(Clinical Focus: This set of values contains diagnoses that are commonly used in the inpatient setting for newborn live born, and born in hospital. This value set is a grouping value set composed of  ICD-10-CM and SNOMED CT value sets.),(Data Element Scope: The intent of this data element is to identify patients who have an active diagnosis of newborn live born. Using the Quality Data Model, this value set is used with \"Diagnosis\" category or the attributes \"Principal Diagnosis\" or \"Diagnoses\" attributes of the \"Encounter\" category.),(Inclusion Criteria: This value set includes ICD-10-CM  and SNOMED CT value sets.),(Exclusion Criteria: None.)"
```

Comment:
Same as above with the double quotes. This aligns better with the other elements, which use single quotes, as well.

```suggestion
Description: "(Clinical Focus: This set of values contains diagnoses that are commonly used in the inpatient setting for newborn live born, and born in hospital. This value set is a grouping value set composed of  ICD-10-CM and SNOMED CT value sets.),(Data Element Scope: The intent of this data element is to identify patients who have an active diagnosis of newborn live born. Using the Quality Data Model, this value set is used with 'Diagnosis' category or the attributes 'Principal Diagnosis' or 'Diagnoses' attributes of the 'Encounter' category.),(Inclusion Criteria: This value set includes ICD-10-CM  and SNOMED CT value sets.),(Exclusion Criteria: None.)"
```